endtoendtests/src/main/java/com/microsoft/azure/functions/endtoend/EventHubTriggerTests.java

```diff
@@ -77,11 +77,40 @@ public void TestEventHubOutputInputOne(
         output.setValue(message);
     }
 
+    @FunctionName("EventHubTriggerAndOutputBinaryCardinalityManyListBinary")
+    public void EventHubTriggerAndOutputBinaryCardinalityManyListBinary(
+            @EventHubTrigger(name = "messages", eventHubName = "test-binary-input-cardinality-many-list-java", connection = "AzureWebJobsEventHubSender_2", dataType = "binary", cardinality = Cardinality.MANY) List<byte[]> messages,
+            @QueueOutput(name = "output", queueName = "test-binary-output-cardinality-many-list-java", connection = "AzureWebJobsStorage") OutputBinding<byte[]> output,
+            final ExecutionContext context
```

Comment:
Does this test work without using the new CLI ?

Reply:
It is not but I commented the integration test, so this should not be called.